### PR TITLE
[WIP] Nomenclature improvements and documentation corrections

### PR DESCRIPTION
**Pull request by Anton-Le**
Modifications in nomenclature of the variables as well as in the documentation.

### CODE REVIEW

File: include/picongpu/particles/shapes/NGP.hpp [hunk 1]
```diff
@@ -58,9 +58,9 @@ namespace shapes
                  *       -
                  */
 
-                bool const below_half = -0.5_X <= x && x < 0.5_X;
+                bool const isWithin_0_5 = -0.5_X <= x && x < 0.5_X;
```
Comment (sbastrakov) [inline include/picongpu/particles/shapes/NGP.hpp]: Perhaps this variable can also be called `isInSupport` ?

File: include/picongpu/particles/shapes/P4S.hpp [hunk 2]
```diff
@@ -101,17 +101,17 @@ struct P4S : public shared_P4S::P4S
              */
             float_X const abs_x = algorithms::math::abs( x );
 
-            bool const below_2nd_radius = abs_x < 1.5_X;
-            bool const below_1st_radius = abs_x < 0.5_X;
+            bool const isWithin_2nd_radius = abs_x < 1.5_X;
+            bool const isWithin_1st_radius = abs_x < 0.5_X;
```
Comment (sbastrakov) [inline include/picongpu/particles/shapes/P4S.hpp]: I think this is okay, would have personally prefered camelCase-conformant `isWithinSecondRadius`. I understand other variable names are also not all in camelCase.

File: include/picongpu/particles/shapes/P4S.hpp [hunk 3]
```diff
@@ -137,12 +137,12 @@ struct P4S : public shared_P4S::P4S
              */
             float_X const abs_x = algorithms::math::abs( x );
 
-            bool const below_max = abs_x < 2.5_X;
+            bool const isWithin_2_5 = abs_x < 2.5_X;
```
Comment (sbastrakov) [inline include/picongpu/particles/shapes/P4S.hpp]: Isn't this also logically `isInSupport` ?

File: include/picongpu/particles/shapes/PCS.hpp [hunk 4]
```diff
@@ -75,16 +75,16 @@ struct PCS : public shared_PCS::PCS
              */
             float_X const abs_x = algorithms::math::abs( x );
 
-            bool const below_1 = abs_x < 1.0_X;
-            bool const below_2 = abs_x < 2.0_X;
+            bool const isWithin_1_0 = abs_x < 1.0_X;
+            bool const isWithin_2_0 = abs_x < 2.0_X;
```
Comment (sbastrakov) [inline include/picongpu/particles/shapes/PCS.hpp]: Is there a reason these names differ from the 1st and 2nd radii in the P4S shape?

**Comment by Anton-Le** [comment]
@sbastrakov 
Regarding the account @PrometheusPi has mentioned it to me yesterday, so I set the name today - hence the change.
I'll have a look at the variable names again, they are a bit confusing. Ideally one would indeed call all `isInSupport` or `argumentIsInSupport`, but the problem is the range. Formally support of a function is the set of values for which it's non-vanishing. If it is defined piecewise in the support doesn't matter.

Perhaps `isInSupport_a_b`, with the meaning of `a` and `b` as given in the commit description would be more meaningful.

File: docs/source/usage/plugins/radiation.rst [hunk 5]
```diff
@@ -15,8 +15,8 @@ Our simulation computes the `Lienard Wiechert potentials <https://en.wikipedia.o
 Variable                       Meaning
 ============================== ================================================================================
 :math:`\vec r_k(t)`            The position of particle *k* at time *t*.
-:math:`\vec \beta_k(t)`        The normalized speed of particle *k* at time *t*.
-                               (Speed divided by the speed of light)
+:math:`\vec \beta_k(t)`        The normalized velocity of particle *k* at time *t*.
```
Comment (psychocoderHPC) [inline docs/source/usage/plugins/radiation.rst]: This change should not be part of this PR, it has nothing todo with the shapes. Could you please push it separate?

**Comment by psychocoderHPC** [comment]
what is the state of this pr?

**Comment by psychocoderHPC** [comment]
What is the state of this PR, it is now nearly a year in the queue.

**Comment by psychocoderHPC** [comment]
@Anton-Le  could you finish this PR

**Comment by psychocoderHPC** [comment]
I set this PR to `draft` because there is no progress.

**Comment by psychocoderHPC** [comment]
@Anton-Le Please open a new PR if you still like to bring these changes into the mainline.
